### PR TITLE
Adding a few more scenarios

### PR DESCRIPTION
**Pull request by mibulgad**
I'm expanding the note in line #269 to clarify why is so important to make sure the platform is allowed in the default enrollment restriction policy for useless scenarios.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@mibulgad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/enrollment-restrictions-set.md [hunk 1]
```diff
@@ -266,7 +266,8 @@ When you create a restriction, it's added to the list just above the default. Yo
 
 
 >[!NOTE]
->Enrollment restrictions are applied to users. For enrollment scenarios that are not user-driven, such as Windows Autopilot self-deploying mode, Intune only enforces the default restrictions targeted to all users.  
+>Enrollment restrictions are applied to users. For enrollment scenarios that are not user-driven such as Windows Autopilot self-deploying mode, Bulk enrollment (WCD), or Azure Virtual desktop, Intune only enforces the default restrictions targeted to all users.
+>For a succesful enrollment, make sure the platform is allowed in the default enrollment restriction policy.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-restrictions-set.md:270]:
```suggestion
>
>For a successful enrollment, make sure the platform is allowed in the default enrollment restriction policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mibulgad, kindly commit the suggested change. Many thanks.

**Comment by PRMerger8** [comment]
@lenewsad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**